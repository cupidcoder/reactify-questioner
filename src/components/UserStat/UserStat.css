.UserStat {
  display: flex;
  margin-top: 1.5rem;
  padding: 1rem;
}
