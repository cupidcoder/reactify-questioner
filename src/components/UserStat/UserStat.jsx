import React from 'react';
import UserStatCard from '../UserStatCard/UserStatCard';
import style from './UserStat.css';

const UserStat = () => {
  const stats = [
    { no: 1, stat: 39, text: 'Questions' },
    { no: 2, stat: 14, text: 'Comments' }
  ];
  return (
    <div className={style.UserStat}>
      {stats.map(stat => <UserStatCard key={stat.no} stat={stat.stat} text={stat.text} />)}
    </div>
  );
};
export default UserStat;
