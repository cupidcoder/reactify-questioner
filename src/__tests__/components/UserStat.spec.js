import React from 'react';
import { shallow } from 'enzyme';
import UserStat from '../../components/UserStat/UserStat';

describe('UserStat', () => {
  it('Should correctly save snapshot of the UserStat', () => {
    const wrapper = shallow(<UserStat />);
    expect(wrapper).toMatchSnapshot();
  });
});
