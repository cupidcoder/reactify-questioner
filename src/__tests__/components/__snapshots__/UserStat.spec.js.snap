// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserStat Should correctly save snapshot of the UserStat 1`] = `ShallowWrapper {}`;
